specs/am/architecture-management-shapes.ttl

```diff
@@ -20,60 +20,72 @@
 
 @prefix oslc_am:    <http://open-services.net/ns/am#> .
 @prefix jazz_am:    <http://jazz.net/ns/dm/linktypes#> .
-
-@base <http://open-services.net/ns/core/shapes/3.0/architecture-management-shape.ttl> .
+@prefix : <http://open-services.net/ns/am/shapes/3.0#> . 
 
 # OSLC Architecutre Management Resource Shapes
 
-<#Resource>
+:
+  a oslc:ResourceShapeConstraints ;
+  dcterms:title "The OSLC Architecture Management(AM) Coonstraints" ;
+  rdfs:label "The OSLC Architecture Management(AM) Coonstraints" ;
```

Comment:
May I suggest "OSLC Architecture Management (AM) Coonstraints"? Note the dropped article and an added space before the abbreviation.

Reply:
yes